components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/ActionExecutorConfig.java

```diff
@@ -44,6 +44,15 @@ public class ActionExecutorConfig {
             "Actions.ActionRequest.ExcludedHeaders.Header";
     private static final String EXCLUDED_PARAMS_IN_ACTION_REQUEST_PROPERTY =
             "Actions.ActionRequest.ExcludedParameters.Parameter";
+    private static final String HTTP_READ_TIMEOUT_PROPERTY = "Actions.HTTPConnections.HTTPReadTimeout";
+    private static final String HTTP_CONNECTION_REQUEST_TIMEOUT_PROPERTY =
+            "Actions.HTTPConnections.HTTPConnectionRequestTimeout";
+    private static final String HTTP_CONNECTION_TIMEOUT_PROPERTY = "Actions.HTTPConnections.HTTPConnectionTimeout";
+    private static final String HTTP_CLIENT_CONNECTION_POOL_SIZE_PROPERTY = "Actions.HTTPClientConnectionPoolSize";
```

Comment:
Let's go with Actions.HTTPConnections. HTTPClientConnectionPoolSize

Or else, we have to update all for below.

Actions.HTTPClient. HTTPReadTimeout
Actions.HTTPClient. HTTPConnectionRequestTimeout
Actions.HTTPClient. HTTPConnectionTimeout
Actions.HTTPClient. HTTPClientConnectionPoolSize

I prefer the second one. This will give us room to onboard different other clients as well

Reply:
@ashanthamara WDYT on the new suggestion

Reply:
+1 for the second suggestion.

Reply:
Updated in